### ISSUE TITLE
Style registration page

### ISSUE THREAD

**Issue by armiantos** [UAlbertaCompEClub/compe-plus#190]
Registration page should be a single column consisting of year, program, and submit/cancel buttons

Integration is done by #189 so you can branch off of this branch using

```sh
git fetch                               # pulls remote branches
git switch armi/integrate-register      # switches to my branch
git switch -c anushka/style-register    # create a new branch based off of mine
```

**Comment by armiantos** [comment]
Since `armi/integrate-register` has been merged, you can just branch off of master

**Comment by armiantos** [comment]
Might be useful to take a look at [Material UI Grid](https://material-ui.com/components/grid/) to style the registration page. [Landing.tsx](https://github.com/UAlbertaCompEClub/compe-plus/blob/master/www/src/routes/unauthenticated/Landing.tsx) is also a good thorough use of how we can use `Grid` to layout the homepage into independent, vertical sections.

**Comment by armiantos** [comment]
Closed by #248
